XpringKit/XRPCurrency.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+/// A currency on the XRP Ledger
+/// - SeeAlso: 
+public struct XRPCurrency: Equatable {
```

Comment:
Is Equatable protocol like operator overloading for comparability?

Reply:
It's a protocol that says we can use the `==` operator.  

It means we have to implement a `==` method (recall that in swift operators are functions), but in turn methods like `XCTAssertEquals` which takes `Equatable` objects magically works. 

You may see we haven't implemented the `==` method. That is because if we use a `struct` (instead of a class) and all our iVars are also `Equatable` then the compiler magically makes a `==` method for us.